390/Serena_Li.Rmd

```diff
@@ -0,0 +1,30 @@
+---
+title: "MP2 Viz"
+author: "Serena Li"
+date: "9/15/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(tidyverse)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+library(dplyr)
+library(readr)
+
+```
+
+```{r}
+stem <- read_csv("tree_main_census/data/census-csv-files/scbi.stem1.csv")
+```
+
+#barplot
+```{r}
+ggplot(stem,aes(x = sp, y=dbh)) +
+  geom_boxplot()+coord_flip()
```

Comment:
Nice job with the `coord_flip`! Makes the species! labels a lot more readable!

Reply:
One thing I'm going to be very strict on however is axes labels and titles. It is really important when analyzing data to give as much as of the data's context as possible. Adding a `labs(x = "blah", y = "blah 2", title = "blah 3")` layer to your plot will go a long way to explaining the context. 

Please add some informative labels. After that, we should be ready to merge.